monai/transforms/lazy/functional.py

```diff
@@ -24,19 +23,15 @@
     kwargs_from_pending,
     resample,
 )
-from monai.utils import LazyAttr
+from monai.utils import LazyAttr, look_up_option
 
 __all__ = ["apply_transforms"]
 
+__override_keywords = {"mode", "padding_mode", "dtype", "align_corners", "resample_mode"}
+
 
 def apply_transforms(
-    data: torch.Tensor | MetaTensor,
-    pending: list | None = None,
-    mode: str | int | None = None,
-    padding_mode: str | None = None,
-    dtype=np.float64,
-    align_corners: bool = False,
-    resample_mode: str | None = None,
+    data: torch.Tensor | MetaTensor, pending: list | None = None, overrides: dict | None = None, **kwargs: Any
```

Comment:
@atbenmurray I changed to keep `**kwargs` here because there are many calls in the dev branch:
```
 result = apply_transforms(pending_result, mode="nearest", padding_mode=mode[1], align_corners=False)[0]
```
also andI think it's convenient to enable this